Directory.Packages.props

```diff
@@ -2,47 +2,69 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <NoWarn>NU1507</NoWarn>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
+  
+  <!-- Global packages (private, build-time packages for all projects) -->
+  <ItemGroup>
+    <GlobalPackageReference Include="Nerdbank.GitVersioning" Version="3.6.133" />
+    <GlobalPackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.507" />
+    <GlobalPackageReference Include="Umbraco.Code" Version="2.0.0" />
+    <GlobalPackageReference Include="Umbraco.GitVersioning.Extensions" Version="0.2.0" />
+  </ItemGroup>
+  
+  <!-- Microsoft packages -->
   <ItemGroup>
-    <PackageVersion Include="Asp.Versioning.Mvc" Version="7.1.0" />
-    <PackageVersion Include="Asp.Versioning.Mvc.ApiExplorer" Version="7.1.0" />
-    <PackageVersion Include="Dazinator.Extensions.FileProviders" Version="2.0.0" />
-    <PackageVersion Include="Examine" Version="3.1.0" />
-    <PackageVersion Include="Examine.Core" Version="3.1.0" />
-    <PackageVersion Include="HtmlAgilityPack" Version="1.11.54" />
-    <PackageVersion Include="K4os.Compression.LZ4" Version="1.3.6" />
-    <PackageVersion Include="MailKit" Version="4.3.0" />
-    <PackageVersion Include="Markdown" Version="2.2.1" />
-    <PackageVersion Include="MessagePack" Version="2.5.129" />
     <PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.0" />
     <PackageVersion Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" />
```

Comment:
Microsoft.CSharp has been replaced by Microsoft.CodeAnalysis.CSharp, which is an updated version (part of https://github.com/dotnet/roslyn/, instead of the archived https://github.com/dotnet/corefx repo).